packages/js/src/ui/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,22 @@
+import { Show } from 'solid-js';
+import { InboxNotification } from '../../../types';
+import { MountNotification } from '../../types';
+import { ExternalElementMounter } from '../ExternalElementMounter';
+import { DefaultNotification } from './DefaultNotification';
+
+type NotificationProps = {
+  notification: InboxNotification;
+  mountNotification?: MountNotification;
+};
+
+export const Notification = (props: NotificationProps) => {
+  return (
+    <Show when={props.mountNotification} fallback={<DefaultNotification notification={props.notification} />}>
+      <ExternalElementMounter
+        mount={(el) => {
+          return props.mountNotification!(el, { notification: props.notification });
```

Comment:
🤓  Make it an oneliner
```suggestion
  props.mountNotification!(el, { notification: props.notification });
```